source/CIGameScene.cpp

```diff
@@ -38,8 +38,17 @@ using namespace std;
 /** Maximum number of stardusts allowed on screen at a time. */
 #define MAX_STARDUST 512
 
-/** Starting value for the countdown */
-#define START_COUNTDOWN -10.0f
+/** Maximum number of layers in planet model */
+#define MAX_PLANET_LAYERS 3
+
+/** Default value for the countdown (inactive) */
+#define INACTIVE_COUNTDOWN -10.0f
```

Comment:
Can you name this something more specific. Same with start countdown. Both these variables should say something about winning